packages/widgetbook/README.md

```diff
@@ -377,6 +377,46 @@ Widgetbook<CupertinoThemeData>(
 
 Both the `scaffoldBuilder` and `useCaseBuilder` can be used to wrap the `Widget` with e.g. a `Scaffold` or some other `Widget` like a `Center`, `Container` or `Padding`.  
 
+# Using Widgetbook for a package
+
+A lot of app projects implement dedicated packages for UI components. These packages often do not define a (desktop) app. Therefore, it's hard to use Widgetbook for these packages. This section will explain how you can get Widgetbook running in such a scenario. 
+
+Lets assume you have set up a monorepo with the following folder structure with two packages `ui_components` and `core_api`:
+
+```
+monorepo
+├─ docs
+├─ packages
+│  ├─ ui_components
+│  ├─ core_api
+```
+
+To create a widgetbook for the `ui_components` package we recommend to create a new folder named `widgetbooks` in the root of the repository and add a flutter (desktop) app named `ui_components_widgetbook`. The folder structure of your repository will look like this:
```

Comment:
for the `ui_components` package, we

Reply:
`widgetbooks` --> why plural?

Reply:
plural because if you have a monorepo you likely have multiple (Flutter) packages and/or apps. Therefore, you will also have multiple widgetbooks.